Jaeger Exporter Compilation error

**Description**

getting below issue with version 1.6.1 compilation with flag -DWITH_JAEGER=ON 


69.18 -- The C compiler identification is GNU 9.4.0
69.24 -- The CXX compiler identification is GNU 9.4.0

 123.6 Scanning dependencies of target opentelemetry_exporter_jaeger_trace
123.6 [ 60%] Building CXX object exporters/jaeger/CMakeFiles/opentelemetry_exporter_jaeger_trace.dir/src/jaeger_exporter.cc.o
124.6 In file included from /usr/src/opentelemetry-cpp/exporters/jaeger/src/http_transport.h:3,
124.6                  from /usr/src/opentelemetry-cpp/exporters/jaeger/src/jaeger_exporter.cc:9:
124.6 /usr/src/opentelemetry-cpp/exporters/jaeger/src/THttpTransport.h:23:8: error: 'bool opentelemetry::v1::exporter::jaeger::THttpTransport::isOpen() const' marked 'override', but does not override
 124.6    23 |   bool isOpen() const override;
 124.6       |        ^~~~~~
 124.7 In file included from /usr/src/opentelemetry-cpp/exporters/jaeger/src/udp_transport.h:6,
 124.7                  from /usr/src/opentelemetry-cpp/exporters/jaeger/src/jaeger_exporter.cc:11:
 124.7 /usr/src/opentelemetry-cpp/exporters/jaeger/src/TUDPTransport.h:31:8: error: 'bool opentelemetry::v1::exporter::jaeger::TUDPTransport::isOpen() const' marked 'override', but does not override
 124.7    31 |   bool isOpen() const override;
 124.7       |        ^~~~~~
 125.7 make[2]: *** [exporters/jaeger/CMakeFiles/opentelemetry_exporter_jaeger_trace.dir/build.make:63: exporters/jaeger/CMakeFiles/opentelemetry_exporter_jaeger_trace.dir/src/jaeger_exporter.cc.o] Error 1
125.7 make[1]: *** [CMakeFiles/Makefile2:913: exporters/jaeger/CMakeFiles/opentelemetry_exporter_jaeger_trace.dir/all] Error 2
125.7 make: *** [Makefile:130: all] Error 2

**Steps to reproduce**

FROM ubuntu:20.04
   "install dependencies"
RUN git clone --recursive https://github.com/open-telemetry/opentelemetry-cpp /usr/src/opentelemetry-cpp \
    && mkdir -p /usr/src/opentelemetry-cpp/build && cd /usr/src/opentelemetry-cpp \
    && git checkout v1.6.1 \
    && cd build \
    && cmake .. -DBUILD_TESTING=OFF -DWITH_JAEGER=ON \
    && cmake --build . --target all \
    && cmake --install . \
    && rm -rf /usr/src/opentelemetry-cpp


**What is the expected behavior?**
compilation should be success

**What is the actual behavior?**
compilation error

**Additional context**
Add any other context about the problem here.

--- a/docs/dependencies.md
+++ b/docs/dependencies.md
@@ -77,7 +77,9 @@ Both these dependencies are listed here:
     framework.
     - `thrift` compiler to generate C++ stubs for IDL data model for Jaeger.
     - `libthrift`  library to generate serialised trace/metrics/log data to be
-      sent to remote Jaeger service.
+      sent to remote Jaeger service. Note: libthrift **0.12.0** doesn't work
+      with this Jaeger exporter. See
+      [#1680](https://github.com/open-telemetry/opentelemetry-cpp/issues/1680).
     - License: `Apache License 2.0`
 
 - [ETW](/exporters/etw)

